lookout/style/typos/templates/scores.md.jinja2

```diff
@@ -0,0 +1,20 @@
+### Typos correction report
+
+| Score type | Accuracy | Precision | Recall | F1-score |
+|-----------:|:---------|:----------|:-------|:---------|
+{% for mode in scores -%}
+| `{{ mode }}` | {{ "%.3f" % scores[mode]["accuracy"] -}}
+| {{ "%.3f" % scores[mode]["precision"] -}}
+| {{ "%.3f" % scores[mode]["recall"] -}}
+| {{ "%.3f" % scores[mode]["f1"] -}} |
```

Comment:
Can I ask you to align numbers with spaces to fit table columns to make not only rendered `.md` file looks good but in a raw text also.